src/test/java/com/teragrep/cfe_39/KafkaConsumerTest.java

```diff
@@ -0,0 +1,184 @@
+package com.teragrep.cfe_39;
+
+import com.teragrep.cfe_39.consumers.kafka.KafkaController;
+import org.apache.avro.file.DataFileReader;
+import org.apache.avro.io.DatumReader;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.common.PartitionInfo;
+import org.apache.kafka.common.TopicPartition;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import com.teragrep.cfe_39.avro.SyslogRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+public class KafkaConsumerTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConsumerTest.class);
+    // Make sure application.properties has consumer.useMockKafkaConsumer=true enabled for Kafka testing.
+
+    // @Test
```

Comment:
Don't comment out tests

Reply:
Solved in commit 65342c2e865912f981dfe82f470cab6884f72cdb